bin/add_users.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import os
+
+# Add the parent directory to sys.path
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+from app.db import get_session
+from app.models.users import Users
+from app.auth.security import get_password_hash
+import logging
+
+
+def add_users(users_list_dict: dict):
+    """
+    This function creates new users in the spun up local database
+
+    Args:
+        users_list_dict: Should contain a list of dictionary
+        usernames and plain text passwords for users
+
+    Logs:
+        Logger warning: If username already exists will raise a
+        warning and ignore adding this user. It was also raise
+        a warning if there is a missing username/password.
+    """
+    with next(get_session()) as session:
+        for user_info in users_list_dict:
+            username = user_info.get("username")
+            password = user_info.get("password")
+            disabled = user_info.get("disabled")
```

Comment:
disabled cannot be `None` so can this read `user_info.get("disabled", False)` please.